app/models/accommodation.rb

```diff
@@ -1,3 +1,7 @@
 class Accommodation < ApplicationRecord
   belongs_to :trip
+
+  validates :name, presence: true
+  validates :address, presence: true
+  validates :check_out, comparison: { greater_than: :check_in }
 end
```

Comment:
I am thinking only the most important fields need to be added

Reply:
Yeah, like expenses, if the user don't want to use that, they can leave it blank